store-implementation/accumulo-store/src/test/java/gaffer/accumulostore/utils/AccumuloTestConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gaffer.accumulostore.utils;
+
+import java.io.File;
+
+public class AccumuloTestConstants {
+
+    public static final File tmpDirectory = new File(System.getProperty("java.io.tmpdir"));
```

Comment:
Please could you change this constant to be in uppercase for consistency - e.g TMP_DIRECTORY
This will be useful for other modules too, so it might be better in a CommonTestConstants class in the CommonUtil module.